examples/api/test/material/text_field/text_field.2_test.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/text_field/text_field.2.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Creates styled text fields', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.TextFieldExamplesApp());
+
+    expect(find.text('TextField Examples'), findsOneWidget);
+    expect(find.byType(TextField), findsNWidgets(2));
+    expect(find.byType(example.FilledTextFieldExample), findsOneWidget);
+    expect(find.byType(example.OutlinedTextFieldExample), findsOneWidget);
+
+    final TextField filled = tester.widget<TextField>(find.descendant(
+      of: find.byType(example.FilledTextFieldExample),
+      matching: find.byType(TextField)));
```

Comment:
Missing comma at the end of `matching`parameter line and the two closing parentheses should be on a new line, see:
```suggestion
    final TextField filled = tester.widget<TextField>(find.descendant(
      of: find.byType(example.FilledTextFieldExample),
      matching: find.byType(TextField),
    ));
```

Reply:
thanks